### DISCUSSION TITLE
Can you download music videos only for existing tracks/albums?

### DISCUSSION THREAD

**Discussion by Adrian-Rasmussen** [RandomNinjaAtk/docker-lidarr-extended#112] [Q&A] [answered]
In Lidarr I have some artists where I don't want all albums/tracks, so I have them unmonitored.
For these artists, can I get the music video downloader to skip those albums/tracks and only download music videos for tracks I have on disk?

**Comment by RandomNinjaAtk** [comment, marked as answer]
It's not really possible because there is no database that associates/links the videos to matching tracks that could be used as a reference.

So at best something could be written to attempt to make a guess by comparing file names. But that would not be very accurate.

Edit: It would be more possible to skip unmonitored artists, which would result in skipping the entire artist. But this is not something that exists at this time.
